js/content/store.js

```diff
@@ -688,7 +691,7 @@ class AppPageClass extends StorePageClass {
             function _addHDControl() {
                 // Add "HD" button and "sd-src" to the video and set definition
                 if (videoControl.dataset.hdSrc) {
-                    videoControl.dataset.sdSrc = videoControl.src;
+                    // videoControl.dataset.sdSrc = videoControl.src;
```

Comment:
If this is the fix, you should at least comment it

Reply:
I believe it just wasn't needed anymore.